packages/cli/src/commands/locations/delete.ts

```diff
@@ -15,13 +17,7 @@ export default class LocationsDeleteCommand extends APICommand {
 		const { args, argv, flags } = this.parse(LocationsDeleteCommand)
 		await super.setup(args, argv, flags)
 
-		const config = {
-			primaryKeyName: 'locationId',
-			sortKeyName: 'name',
-		}
-		const id = await selectFromList(this, config, args.id,
-			async () => await this.client.locations.list(),
-			'Select location to delete.')
+		const id = await chooseLocation(this, args.id)
```

Comment:
👍 These are a great refactor and make the `run` more readable and quick to understand.

Reply:
thanks